src/05-led-roulette/my-solution.md

```diff
@@ -53,8 +53,9 @@ Binary size is something we should always keep an eye on! How big is your soluti
 that using the `size` command on the release binary:
 
 ``` console
-$ cargo size -- target/thumbv7em-none-eabihf/{debug,release}/led-roulette
-target/thumbv7em-none-eabihf/debug/led-roulette  :
+$ # equivalent to size target/thumbv7em-none-eabihf/debug/led-roulette
+$ cargo size --bin led-roulette -- -A
+led-roulette  :
```

Comment:
Leads to the following error for me:
```
cargo size --bin led-roulette -- -A
   Compiling led-roulette v0.1.0 (file:///home/caemor/git/discovery/src/05-led-roulette)                
error: language item required, but not found: `eh_personality`

error: aborting due to previous error

error: Could not compile `led-roulette`.

To learn more, run the command again with --verbose.
```
So I still needed to use 
`cargo size --bin led-roulette --target thumbv7em-none-eabihf -- -A`

cargo size (cargo-binutils) is installed in version 0.1.2.